spring-integration-core/src/main/java/org/springframework/integration/handler/AbstractMessageHandler.java

```diff
@@ -174,7 +170,7 @@ public void handleMessage(Message<?> message) {
 						.tag("name", getComponentName() == null ? "unknown" : getComponentName())
 						.tag("result", "failure")
 						.tag("exception", e.getClass().getSimpleName())
-						.description("Subflow process time")
+						.description("Send processing time")
```

Comment:
It's `AbstractMessageHandler`, so, I guess `Handle processing time` would be better. Or something similar, but definitely not `send`. The same is applied for the `sendTimer()` method name below.

Also if you apply my polishing in the `AbstractMessageChannel`, I think similar refactoring should be done in this class as well.

Reply:
@artembilan Note that the same timer name is used by handlers and channels (which is intentional). Monitoring systems that receive description text will only display one of them and we don't want that to be non-deterministic. We should maintain a unified description text between handlers and channels whatever we do.

Reply:
Not sure why do you think it the same matter.

When we use an `ExecutorChannel` or `FluxChannel` (or even `QueueChannel`), our `send` is performed on one, calling thread. But `handle` will happen definitely in a different thread. That is only a case of `DirectChannel` that `send` and `handle` overlap. 

So, now you are confusing me even more. IMO we indeed need a separate `send`, `handle` and `receive` metrics.
What's wrong with that?

Thanks

Reply:
In discussion with Jon, given the nature of how the back-ends work, we need a single metric name; we decided on `send` after struggling for some time with `flow`, `subflow`, `process` etc.

Reply:
Just great! No any clues on the matter throughout the PR...

How about to mention this in the Docs as well?
At least make me happy do not ask similar questions in the future.

Thanks and sorry for such a noise during review.

Reply:
It's not noise; they are valid questions.

Reply:
We can enhance the docs later; we will work with @jkschneider and clean up cafe sample app to create some sexy screen shots which will explain better why the naming strategy was chosen.

Reply:
OK. So, what we do with this PR then?
Wait for screenshots?
Shell I polish the mentioned code and merge?
Anything else ?

Thanks

Reply:
Screenshots can follow after 5.0.3; let's go ahead (with your polishing) and get the 5.0.3.RELEASE train rolling 😄